src/contracts/SmartContract.sol

```diff
@@ -29,31 +31,42 @@ contract SmartContract {
     uint public extraTime;
     address public helperID;
     uint public initialTimestamp;
+    Status public contractStatus = Status.Setup;
 
     mapping(uint => PuzzlePart) public puzzleParts;
     uint amountOfPuzzleParts;
```

Comment:
Should this be initialised to 0 explicitly?

Reply:
No, but again, feels better for readability